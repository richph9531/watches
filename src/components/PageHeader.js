import React from 'react';

import classes from '../styles/PageHeader.module.css';

const PageHeader = () => (
    <div className={classes.PageHeader}>
      <img
        src={`${process.env.PUBLIC_URL}/media/AmazonLogo.svg`}
        alt='Amazon Logo'/>
    </div>
);

export default PageHeader;
