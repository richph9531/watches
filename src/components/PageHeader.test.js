import React from 'react';

import { render } from '@testing-library/react';
import PageHeader from './PageHeader';

test('Page header renders correctly', () => {
  const renderedElement = render(<PageHeader />);
  expect(renderedElement.baseElement).toMatchSnapshot();
});
