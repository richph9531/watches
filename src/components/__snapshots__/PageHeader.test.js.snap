// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page header renders correctly 1`] = `
<body>
  <div>
    <div
      class="PageHeader"
    >
      <img
        alt="Amazon Logo"
        src="/media/AmazonLogo.svg"
      />
    </div>
  </div>
</body>
`;
